### DISCUSSION TITLE
Class definition for 'ComputeCrystalPlasticityThermalEigenstrain'

### DISCUSSION THREAD

**Discussion by abarun22** [idaholab/moose#25389] [Q&A General]
Could someone please direct me to the page where i can find the class definition for the above class?

**Comment by GiudGiud** [comment]
This page?
https://mooseframework.inl.gov/source/materials/crystal_plasticity/ComputeCrystalPlasticityThermalEigenstrain.html
or the doxygen one?
I cant find the doxygen but this one is the base class and most of the APIs shared
https://mooseframework.inl.gov/docs/doxygen/modules/classComputeCrystalPlasticityEigenstrainBase.html

**Comment by abarun22** [comment]
Yeah, I was looking for the first one. Thank you.
